security/cyrus-sasl2/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup               muniversal 1.0
 PortGroup               legacysupport 1.0
 
 github.setup            cyrusimap cyrus-sasl 2.1.27 cyrus-sasl-
-revision                3
+revision                4
```

Comment:
It would be nice to bump the revision to 5 and remove the platform darwin 20 block below that increments the revision.